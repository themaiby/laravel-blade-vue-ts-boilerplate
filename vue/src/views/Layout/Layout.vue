<template>
  <span>I'm an layout</span>
</template>

<script lang="ts">
import { Component, Vue } from 'vue-property-decorator';

@Component({ name: 'Layout' })
export default class Layout extends Vue {}
</script>

<style lang="scss" src="style.scss"></style>
