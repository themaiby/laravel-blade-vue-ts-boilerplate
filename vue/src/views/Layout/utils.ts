/** utils file for layout only */
