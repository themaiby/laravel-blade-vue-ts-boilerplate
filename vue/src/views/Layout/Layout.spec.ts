import Layout from '@/views/Layout/Layout.vue';
import { mount } from '@vue/test-utils';

describe('Example', () => {
  const layoutComponentWrapper = mount(Layout);

  it('should show render snapshot', () => {
    expect(layoutComponentWrapper).toMatchSnapshot();
  });
});
