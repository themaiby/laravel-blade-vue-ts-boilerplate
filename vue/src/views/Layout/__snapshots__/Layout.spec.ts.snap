// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example should show render snapshot 1`] = `<span>I'm an layout</span>`;
